settings.py

```diff
@@ -21,6 +21,6 @@
 
 USE_I18N = True
 
-USE_TZ = False
+USE_TZ = "Europe/Kiev"
```

Comment:
The `USE_TZ` setting should be a boolean value (`True` or `False`), not a string. Change it to `True` to enable timezone support or `False` to disable it.